arrays-intermediate.js

```diff
@@ -31,16 +37,24 @@ function findFirstStringStartingWithLetter(letter, strings) {
 **************************************************************/
 function isPresentIncluded(presentName, presents) {
   //TODO: Add your code here
+  let presentIncluded = false
+  let arr = presents.map((present) => {
```

Comment:
Instead of using `map ` in this function, you could've used `foreach`, because you're not aiming to create a new array, you're just check if the present is there and changing the value of the `presentIncluded` inside the loop.